sdk/cosmos/azure-cosmos/README.md

```diff
@@ -255,7 +255,7 @@ or contact [opencode@microsoft.com][coc_contact] with any additional questions o
 <!-- LINKS -->
 [source_code]: src
 [cosmos_introduction]: https://docs.microsoft.com/en-us/azure/cosmos-db/
-[api_documentation]: https://azuresdkdocs.blob.core.windows.net/$web/java/azure-cosmos/4.0.1-beta.3/index.html
+[api_documentation]: https://azuresdkdocs.blob.core.windows.net/$web/java/azure-cosmos/4.0.1/index.html
```

Comment:
Anyways to references to latest?

Reply:
https://azuresdkdocs.blob.core.windows.net/$web/java/azure-cosmos/latest/index.html should work

Reply:
Good catch @kirankumarkolli  - I have updated to the latest.